packages/cli/src/commands/link.ts

```diff
@@ -4,10 +4,29 @@ import * as readline from "node:readline/promises";
 import { readFile, writeFile } from "node:fs/promises";
 import { GLOBAL_CONFIG_FILE, LOCAL_CONFIG_FILE } from "../config";
 import { ensureFileInPath } from "../fs-utils";
+import type {
+  CommonYargsArgv,
+  StrictYargsOptionsToInterface,
+} from "./yargs-types";
 
-export const link = async () => {
-  const rl = readline.createInterface({ input: stdin, output: stdout });
-  const shareLink = await rl.question(`Paste share link (with build access): `);
+export const linkOptions = (yargs: CommonYargsArgv) =>
+  yargs.option("link", {
+    alias: "l",
+    type: "string",
+    describe: "Link to a webstudio project",
+  });
+
+export const link = async (
+  options: StrictYargsOptionsToInterface<typeof linkOptions> | { link?: string }
+) => {
+  let shareLink: string;
+  if (options.link) {
+    shareLink = options.link;
+  } else {
+    const rl = readline.createInterface({ input: stdin, output: stdout });
+    shareLink = await rl.question(`Paste share link (with build access): `);
```

Comment:
view access should be enough?

Reply:
- view access is enough, so not worth mentioning at all, because its the lowest permission level anyways
- Better wording: `Please paste a link provided in the "Share" dialog in the Builder`

Reply:
updated wording again